code/modules/reagents/reagent_containers/food/snacks/cultural/human.dm

```diff
@@ -417,3 +462,115 @@
 	reagent_data = list(/singleton/reagent/nutriment = list("blue raspberry" = 5, "white chocolate" = 3))
 	drop_sound = 'sound/items/drop/glass.ogg'
 	pickup_sound = 'sound/items/pickup/glass.ogg'
+
+
+// Eridani
+
+/obj/item/reagent_containers/food/snacks/bowl
+	name = "a bowl of item"
+	desc = "If you're seeing this, something has gone wrong D:"
+	icon = 'icons/obj/item/reagent_containers/food/cultural/human.dmi'
+	icon_state = "puffpuffbowl"
+	trash = /obj/item/trash/snack_bowl
+	var/vendingobject = /obj/item/reagent_containers/food/snacks/puffpuff
+	reagent_data = list(/singleton/reagent/nutriment = list("fried dough" = 10, "ginger" = 4))
+	bitesize = 4
+	reagents_to_add = list(/singleton/reagent/nutriment = 24)
+	var/unitname = "item"
+
+/obj/item/reagent_containers/food/snacks/bowl/puffpuffs
+	name = "puff-puff bowl"
+	desc = "A bowl of puffy dough balls. Much like donut balls except pan fried, chewier, and often served savory, not just sweet. It originates in Nigeria, but this is the Eridani variant, which is made with ginger instead of pepper."
+	bitesize = 4
+	reagents_to_add = list(/singleton/reagent/nutriment/ = 24)
+	unitname = "puff-puff"
+	filling_color = "#bb8a41"
+
+/obj/item/reagent_containers/food/snacks/bowl/attack_hand(mob/user as mob)
+	var/obj/item/reagent_containers/food/snacks/returningitem = new vendingobject(loc)
+	returningitem.reagents.clear_reagents()
+	reagents.trans_to(returningitem, bitesize)
+	returningitem.bitesize = bitesize/2
+	user.put_in_hands(returningitem)
+	if (reagents && reagents.total_volume)
+		to_chat(user, "You take a [unitname] from the plate.")
```

Comment:
```suggestion
		to_chat(user, "You take \a [unitname] from the plate.")
```

Reply:
Hm? I don't understand, why add a slash there?

Reply:
https://secure.byond.com/docs/ref/index.html#/DM/text/macros

\a, \an,   // insert 'a(n)' or 'some' (if needed)

It's a text macro to avoid duplicating the article